python/hail/methods/impex.py

```diff
@@ -548,7 +552,95 @@ def grep(regex, path, max_count=100):
            contig_recoding=nullable(dictof(strlike, strlike)))
 def import_bgen(path, tolerance=0.2, sample_file=None, min_partitions=None, reference_genome=None,
                 contig_recoding=None):
-    rg = reference_genome if reference_genome else Env.hc().default_reference
+    """Import BGEN file(s) as matrix table.
+
+    Warning
+    -------
+
+    A BGEN file must have a ``.idx`` file which can be generated by
+    :func:`.index_bgen`
```

Comment:
Punctuation: 
```
A BGEN file must have a ``.idx`` file, which can be generated by
:func:`.index_bgen`.
```
Also, I don't understand what this is saying. Should it be ".idx file extension"?